raft/server.cc

```diff
@@ -626,8 +637,8 @@ future<> server_impl::wait_for_entry(entry_id eid, wait_type type, seastar::abor
     }
     SCYLLA_ASSERT(inserted);
     if (as) {
-        it->second.abort = as->subscribe([it = it, &container] () noexcept {
-            it->second.done.set_exception(request_aborted());
+        it->second.abort = as->subscribe([it = it, &container]() noexcept {
+            it->second.done.set_exception(request_aborted(format("Abort requested while waiting for entry with idx: {}, term: {}", it->first, it->second.term)));
```

Comment:
I think I remember an earlier comment (which was a good one) that this message and the one above should be different so we know where the exception originated.

How about in the first message above (inside `if (as && as->abort_requested())`), say "Abort requested before waiting ..." (i.e. instead of **while** say **before**) -- because indeed we still haven't created the promise-future pair by that time, we only performed some preliminary checks, we didn't start the wait itself.